articles/quickstart/spa/angular2/04-authorization.md

```diff
@@ -115,11 +109,13 @@ export const ROUTES: Routes = [
 ];
 ```
 
-The guard implements the `CanActivate` interface which requires a method called `canActivate` in the service. This method returns `true` if the user is authenticated and `false` if not. It also navigates the user to the home route if they aren't authenticated.
+The guard implements the `CanActivate` interface which requires a method called `canActivate` in the service. This method returns `true` if the user is authenticated and `false` if they are not. It also navigates the user to the home route if they are not authenticated.
```

Comment:
The interface is from the Angular core, so we can't change it (https://angular.io/api/router/CanActivate)